test/benchmark-queue-work.c

```diff
@@ -0,0 +1,47 @@
+#include "task.h"
```

Comment:
Can you add the "Copyright libuv contributors. All rights reserved." license boilerplate here? You can copy it from e.g. test/test-env-vars.c.